projects/arctic_lstm_speculator/llama3.1-70b.yaml

```
@@ -0,0 +1,50 @@
+type: arctic-lstm-speculator
+micro_batch_size: 16
+gradient_accumulation_steps: 16
+train_iters: 3000
+model:
+  name_or_path: meta-llama/Llama-3.1-70B-Instruct
+  disable_activation_checkpoint: true
+  method: sum_lstm
+  proj_dim: "4096"
+  emb_dim: "4096"
+  speculator_width: "4096"
```

```suggestion
  proj_dim: 4096
  emb_dim: 4096
  speculator_width: 4096
```

These should be numeric, not strings